### DISCUSSION TITLE
grafica

### DISCUSSION THREAD

**Discussion by CobraOfficial01** [iisgiua/giuaschool#677] [Idee e proposte]
salve è possibile mettere una grafica moderna all'interno del registro

**Comment by Giuppimack65** [comment]
mi sa che stanno già lavorando a questo

**Comment by trinko** [comment]
> salve è possibile mettere una grafica moderna all'interno del registro

Se vuoi, puoi modificare i template e il file CSS e puoi modificare tutta la grafica.
